README.md

```diff
@@ -27,7 +22,8 @@ sudo python setup.py install
 Or install from pip
 
 ```
-pip install pynuodb
+sudo pip install pynuodb
+sudo pip install --upgrade pynuodb
```

Comment:
Also I'm not sure what other people think, but I don't think we should include `sudo` here. If you use virtualenv you wouldn't want that, and I think it's reasonable to expect people to know when to use it. But maybe I'm in the minority here.